norse/__init__.py

```diff
@@ -3,6 +3,11 @@
 
 from . import benchmark, dataset, task
 
+import torch
+from pathlib import Path
+
+torch.ops.load_library(Path(__file__).resolve().parent.parent / "norse_op.so")
```

Comment:
Does this still work if we don't build the C++ library? Do we want to make it mandatory to build it?

Reply:
Great point. I found a solution where we basically set a flag to indicate whether the ops modules have been loaded. I tested whether it works both with and without. In the future, though, I think we should do something like the PyTorch guys, where modules are basically mirrored in Python and in compiled code.